src/js/edu-benefits/5490/helpers.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
Can we delete everything but the transform in here? If we need to reuse any of the other data, it should be in a file that's shared between the forms.